app/controllers/followings_controller.rb

```diff
@@ -19,7 +19,7 @@ def tags
                else
                  relation.where(explicit_points: (0...))
                end
-    relation = relation.order(explicit_points: :desc)
+    relation = relation.order(created_at: :desc)
```

Comment:
Ordering by explicit points seems futile because the explicit points will all just be -1 from now onwards. Perhaps we could update by `updated_at` which I think may show the most recently updated card. I want to think sit on this a bit more together with making sure that its aligned with the pagination. I will improve on it in a subsequent PR.

Reply:
For what it's worth, as a user, I think I expect the sorting to be alphabetical, but I can see how created/updated makes sense, as well.